neurodamus/io/sonata_config.py

```diff
@@ -371,28 +369,21 @@ def parsedStimuli(self):
         }
         module_translation = {"seclamp": "SEClamp", "subthreshold": "SubThreshold"}
 
-        stimuli = {}
-        for name in self._sim_conf.list_input_names:
+        stimuli = []
+        # TODO: loop over self._sim_conf.list_input_names,
```

Comment:
do you still have this to do?

Reply:
yes, after we update the SONATA spec and libsonata API